x-pack/filebeat/module/misp/threat/config/input.yml

```diff
@@ -10,6 +10,7 @@ interval: {{ .interval }}
 json_objects_array: {{ .json_objects_array }}
 pagination: {{ .pagination }}
 url: {{ .url }}
+ssl: {{ .ssl }}
```

Comment:
I don't see `ssl` declared in the module's manifest, maybe it missed it. It needs to be declared otherwise the template evaluation should fail. I recently updated Filebeat to be more strict about the usage of undefined variables.

Reply:
Also please make sure the parameters are documented on the MISP module page (in case it's not there already).

Reply:
> I don't see `ssl` declared in the module's manifest, maybe it missed it. It needs to be declared otherwise the template evaluation should fail. I recently updated Filebeat to be more strict about the usage of undefined variables.

Is this change backported to 7.x. I have this fix in my 7.x branch and it is running in GCP for MISP without any problems. 

I can add the empty default SSL config in manifest if it makes a difference. Thanks.

Reply:
> Also please make sure the parameters are documented on the MISP module page (in case it's not there already).

Yes, it is documented already.

Reply:
My changes is not yet backported to 7.x. It's in https://github.com/elastic/beats/pull/14712. It would only cause an issue when `.ssl` is not defined anywhere. So it would be good to always have the module params defined in the manifest.

Reply:
Not a problem. I will add the default empty SSL config. Thanks